python/test/unit/io/test_XDMF.py

```diff
@@ -65,6 +66,18 @@ def worker_id(request):
         return 'master'
 
 
+@pytest.fixture
```

Comment:
Why is this fixture? It is not used as fixture, just as a method.

Reply:
Indeed.. I forgot to remove the decorator.